.bin/create-rust-webpack.js

```diff
@@ -32,13 +32,18 @@ if (!isPresent("git --version")) {
   process.exit(1);
 }
 
+const args = process.argv.slice(2);
```

Comment:
Won't this crash, if the arguments are not expected.

Reply:
oops!